front-end/src/components/Header/Header.js

```diff
@@ -1,11 +1,38 @@
-import React from 'react';
-import './style.scss';
+import React from "react";
+import "./style.css";
 
 export default function Header() {
   return (
     <header>
-      Header components
+      <div className="header-logo">
+        <img src={"./images/Logo.png"} />
+      </div>
+      <div className="header-search">
+        <div className="header-search__bar">
+          <img src={"./images/Search.png"} />
+          <input
+            className="search__bar__input"
+            placeholder="Search..."
+            size="10"
+          />
+        </div>
+      </div>
+      <div className="header-ellipse-icon">
+        <img src={"./images/Ellipse 176.png"} />
```

Comment:
@Taiga17 -san
The image can not have blank in file name and the image need import after import React.
Because if do not import when we deploy, we can not see images.